src/App.js

```diff
@@ -1,11 +1,9 @@
 import React from "react";
 import "./App.css";
+import Display from "./components/DisplayComponents/Display";
 // STEP 4 - import the button and display components
```

Comment:
The Display is not the parent component of the other components. App is. Display is the single grey component that houses the number output. You should be importing everything into App, not Display.